### PR TITLE
expand the regex for various link expansions that twitter is using now

### PR DESCRIPTION
**Pull request by maxtaco**

### CODE REVIEW

**Comment by maxtaco** [comment]
cc: @mlsteele

**Comment by maxtaco** [comment]
@malgorithms do you want to take a look at this too?

**Comment by maxtaco** [comment]
Matches these 3:

```
Checking tweet 'Verifying myself: I am alice3 on http://Keybase.io . S2TeC15SV3QvKDnOKju4fDgyoen5uK2O53qI / https://keybase.io/alice3/sigs/S2TeC15SV3QvKDnOKju4fDgyoen5uK2O53qI …
```
```
Verifying myself: I am alice3 on https://t.co/VsGHLpRPbk. S2TeC15SV3QvKDnOKju4fDgyoen5uK2O53qI / https://t.co/H6XiZPiVLO'
```

```
Checking tweet 'Verifying myself: I am alice3 on Keybase.io. S2TeC15SV3QvKDnOKju4fDgyoen5uK2O53qI / https://keybase.io/alice3/sigs/S2TeC15SV3QvKDnOKju4fDgyoen5uK2O53qI …
```

**Comment by oconnor663** [comment]
I'm gonna review and land this PR while @maxtaco is in a call.

**Comment by maxtaco** [comment]
My tester:

```coffee
proof_text_check = "Verifying myself: I am alice3 on Keybase.io. S2TeC15SV3QvKDnOKju4fDgyoen5uK2O53qI /"
rxx_text = proof_text_check.replace(" on Keybase.io.", " on (Keybase.io|https://t\\.co/\\S*|http://Keybase\\.io\\s)\\.")
rxx = new RegExp ("^" + rxx_text + ".*")
console.log rxx
t = 'Verifying myself: I am alice3 on https://t.co/VsGHLpRPbk. S2TeC15SV3QvKDnOKju4fDgyoen5uK2O53qI / https://t.co/H6XiZPiVLO'
t2 = 'Verifying myself: I am alice3 on http://Keybase.io . S2TeC15SV3QvKDnOKju4fDgyoen5uK2O53qI / https://keybase.io/alice3/sigs/S2TeC15SV3QvKDnOKju4fDgyoen5uK2O53qI …'
console.log rxx
console.log t.match rxx
console.log t2.match rxx
```

File: src/scrapers/twitter.iced [hunk 1]
```diff
@@ -280,15 +280,19 @@ exports.TwitterScraper = class TwitterScraper extends BaseScraper
     inside = ws_normalize inside
     proof_text_check = ws_normalize proof_text_check
 
-    @log "+ Checking tweet '#{inside}' for signature '#{proof_text_check}'"
+    rxx_text = proof_text_check.replace(" on Keybase.io.", " on (Keybase.io|https://t\\.co/\\S*|http://Keybase\\.io\\s)\\.")
```
Comment (oconnor663) [inline src/scrapers/twitter.iced]: I just took a look at Taco von Taco's latest proof in my own browser, and it looks like the "space thing" at the end that they're printing is the U+00A0 nonbreaking space. Go figure.

**Comment by maxtaco** [comment]
aha, that explains it!

File: src/scrapers/twitter.iced (hunk 1 above)
Comment (maxtaco) [reply src/scrapers/twitter.iced]: aha that explains it!

**Comment by oconnor663** [comment]
Just tested by hand, LGTM.

**Comment by chriscpritchard** [comment]
That was really quick, thank you!

**Pull request merged**